### PR TITLE
Fix spacebar switching

### PR DESCRIPTION
**Pull request by rasaford**
DEPENDS ON: https://github.com/Roboy/TeleportVR/pull/45

Fixes various timing issues when scene switching using `space` or controller select. 
Only the last commit in this PR is relevant, as the previous ones are from #45.

### CODE REVIEW

File: Assets/InputDevices/Scripts/InputManager.cs [hunk 1]
```diff
@@ -1,317 +1,321 @@
...
-
-                    //if (Training.TutorialSteps.Instance.currentStep == Training.TutorialSteps.TrainingStep.LEFT_HAND)
-                    //{
-                    //    if (controllerLeft[0].TryGetFeatureValue(UnityEngine.XR.CommonUsages.triggerButton, out btn) &&
-                    //        btn)
-                    //    {
-                    //        Training.TutorialSteps.Instance.CorrectUser("grip");
-                    //    }
-                    //}
+﻿using System.Collections;
```
Comment (missxa) [inline Assets/InputDevices/Scripts/InputManager.cs:1]: why does this whole file show as diff?

**Review by missxa — COMMENTED** [review]
what happened to the original HUD prefab? why did you delete it?

**Comment by rasaford** [comment]
The prefab shows as deleted, as I renamed it in #45. This was due to the fact that the previous name was not longer `HUDSlerpOrigin` was no longer accurate, as only the constanly visible parts slerp with the Camera. 
The pause menu however, only tracks the y-axis rotation.

File: Assets/InputDevices/Scripts/InputManager.cs (hunk 1 above)
Comment (rasaford) [reply Assets/InputDevices/Scripts/InputManager.cs:1]: My fault. I changed the line endings in Visual Studio, as it had mixed ones in there. Now it's `CRLF` everywhere.

**Pull request merged**